citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.jsp

```diff
@@ -163,9 +228,13 @@ $(function() {
 function resetAuthorCreationModal() {
 	$("#firstNameAuthor").val("");
 	$("#lastNameAuthor").val("");
-	$("#affiliationTemplate").find("input").val("");
+	$("#affiliations").html("<div id='affiliationTemplate' class='form-group'><label for='affiliationAuthor'>Affiliation:</label><input type='text' class='form-control' placeholder='Affiliation'></div>");
```

Comment:
I assume you're getting rid of the clones of affiliation inputs? Just give them all a class and then remove them by their class name.